package-lock.json

```diff
@@ -3579,7 +3579,7 @@
     },
     "@types/filewriter": {
       "version": "0.0.28",
-      "resolved": "https://registry.npmjs.org/@types/filewriter/-/filewriter-0.0.28.tgz",
```

Comment:
Can `package-lock.json` be removes as we didn't changed `package.json`

Reply:
fixed